schema_cache.rst

```diff
@@ -89,7 +89,12 @@ See the section :ref:`schema_reloading` to solve this issue.
 Schema Cache Reloading
 ----------------------
 
-To refresh the cache without restarting the PostgREST server, send the server process a SIGUSR1 signal:
+There are two ways of refreshing the cache without restarting the PostgREST server: using Unix signals or sending a database notification.
+
+Unix Signals
+~~~~~~~~~~~~
+
+Send a SIGUSR1 signal to the server process.
```

Comment:
My bad about this, I've now realized that this division doesn't make sense because SIGUSR2 would go in another page. So let's go back to the previous paragraph.

```suggestion
To refresh the cache without restarting the PostgREST server, send the server process a SIGUSR1 signal.
```